packages/app/components/creator-channels/components/message-item.tsx

```diff
@@ -176,7 +177,10 @@ export const MessageItem = memo(
 
               <View tw="flex-row items-center">
                 <Text tw={["text-xs text-gray-700 dark:text-gray-200"]}>
-                  {formatDateRelativeWithIntl(channel_message.created_at)}
+                  {formatDateRelativeWithIntl(
+                    channel_message.created_at,
+                    !isMobileWeb()
```

Comment:
@hirbod this param is false by default, and here i set the desktop only. so they will not have much effect.